acceptancetests/assess_bundle_export.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+""" Test Export Bundle functionality.
+
+  - Exporting current model configuration feature is verified.
+  - Deploy mediawikisimple bundle.
+  - verify if the bundle is created and the content of the file as well by deploying to another model.
+"""
+
+from __future__ import print_function
+
+import argparse
+import glob
+import logging
+import sys
+import os
+import json
+import subprocess
+
+from deploy_stack import (
+    BootstrapManager,
+)
+from jujucharm import (
+    local_charm_path
+)
+from utility import (
+    add_basic_testing_arguments,
+    configure_logging,
+    JujuAssertionError,
+)
+
+
+__metaclass__ = type
+model = "mymodel"
+TEST_MODEL = 'test-tmp-env'
+
+log = logging.getLogger("assess_bundle_export")
+
+def assess_bundle_export(client, args):
+    bundle_source = local_charm_path('mediawiki-simple.yaml',
+                                     repository=os.environ['JUJU_REPOSITORY'],
+                                     juju_ver='2')
+    client.deploy(bundle_source)
+    client.wait_for_started()
+    client.juju('export-bundle', ('--filename', model))
+    log.info("Deploying {}".format("mediawiki-simple bundle..."))
+
+    if not os.path.exists(model+".yaml"):
+        raise JujuAssertionError('export bundle command failed to create bundle file.')
+
+    new_client = add_model(client)
+    new_client.deploy(model+".yaml")
+    new_client.wait_for_started()
+    old_model = get_current_model(client)
+    if old_model != TEST_MODEL:
+        raise JujuAssertionError('Models mismatch : Bundle error.')
+
+    os.remove(model+".yaml")
+
+
+
+def add_model(client):
```

Comment:
These model methods are duplicated code - if used in more than one place they should be put into a helper models.py